docs/modules/ROOT/pages/hazelcast-platform-operator-jet.adoc

```diff
@@ -0,0 +1,339 @@
+= Run a Data Pipeline Using Jet with Hazelcast Platform Operator
+:page-layout: tutorial
+:page-product: operator
+:page-categories: Cloud Native
+:page-lang: go, java, node, python
+:page-enterprise: true
+:page-est-time: 20 mins
+:github-directory: https://github.com/hazelcast-guides/hazelcast-platform-operator-jet
+:description: Learn how to run a data pipeline in Hazelcast using Jet.
+
+{description}
+
+== Context
+In this tutorial, you'll do the following:
+
+- Build and deploy a custom Jet pipeline implementation to cloud storage.
+
+- Deploy Hazelcast with the Jet engine configured.
+
+- Start the Jet pipeline
+
+- Check the Jet pipeline.
+
+- Cancel the Jet pipeline.
+
+== Before You Begin
+
+You need the following:
+
+* Running https://kubernetes.io/[Kubernetes] cluster
+* Kubernetes command-line tool, https://kubernetes.io/docs/tasks/tools/#kubectl[kubectl]
+* Maven command-line tool, https://maven.apache.org/download.cgi[mvn]
+* Deployed xref:operator:ROOT:index.adoc[Hazelcast Platform Operator]
```

Comment:
```suggestion
* Deployed xref:operator:ROOT:index.adoc[Hazelcast Platform Operator version 5.3 or above]
```
I think it would be good to add this so that users don't get halfway through and find out they are using the right version.

Reply:
Required version is for Hazelcast not for the operator.

Reply:
Sorry for misleading information. You are right we need to add version requirement to operator but not 5.3 it should be 5.8. I'll update it